apps/easy-messe/components/AboutUs/statisticMass.tsx

```diff
@@ -0,0 +1,39 @@
+import { theme } from "@easy-messe/libs/theme";
+import { Box, Typography } from "@mui/material";
+import { statisticMasses } from "./AboutUs";
+import { useIntl } from "react-intl";
+import { shortenNumber } from "@easy-messe/libs/utils"
+
+interface statisticProps {
```

Comment:
- [ ] The props interface should match the component name.
- [ ] interfaces should be written in PascalCase

```suggestion
interface StatisticMassProps{
```